packages/store/src/workspace/page.ts

```diff
@@ -106,6 +106,18 @@ export class Page extends Space<PageData> {
     return Array.isArray(this._root) ? this._root[1] : null;
   }
 
+  /** @internal used for getting surface block elements for phasor */
+  get ySurfaceContainer() {
```

Comment:
Here looks tricky, but I didn't find a better way to provide a `yContainer` to the `SurfaceContainer` that manages the canvas context. The `SurfaceContainer` that consumes this YMap is driven by YEvent updates, so this is a must-have.